[2022-08-23 11:58:40,132] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: raizen_lab_01.print_world scheduled__2021-10-21T02:00:00+00:00 [queued]>
[2022-08-23 11:58:40,168] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: raizen_lab_01.print_world scheduled__2021-10-21T02:00:00+00:00 [queued]>
[2022-08-23 11:58:40,169] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-23 11:58:40,169] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-08-23 11:58:40,171] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-23 11:58:40,210] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): print_world> on 2021-10-21 02:00:00+00:00
[2022-08-23 11:58:40,220] {standard_task_runner.py:52} INFO - Started process 188 to run task
[2022-08-23 11:58:40,231] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'raizen_lab_01', 'print_world', 'scheduled__2021-10-21T02:00:00+00:00', '--job-id', '2429', '--raw', '--subdir', 'DAGS_FOLDER/raizen_lab_01.py', '--cfg-path', '/tmp/tmp7t1aq_80', '--error-file', '/tmp/tmpzkznbp2m']
[2022-08-23 11:58:40,233] {standard_task_runner.py:80} INFO - Job 2429: Subtask print_world
[2022-08-23 11:58:40,473] {task_command.py:371} INFO - Running <TaskInstance: raizen_lab_01.print_world scheduled__2021-10-21T02:00:00+00:00 [running]> on host 9f4b06248101
[2022-08-23 11:58:40,713] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=raizen_lab_01
AIRFLOW_CTX_TASK_ID=print_world
AIRFLOW_CTX_EXECUTION_DATE=2021-10-21T02:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-21T02:00:00+00:00
[2022-08-23 11:58:40,717] {logging_mixin.py:115} INFO - world
[2022-08-23 11:58:40,718] {python.py:173} INFO - Done. Returned value was: None
[2022-08-23 11:58:40,824] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=raizen_lab_01, task_id=print_world, execution_date=20211021T020000, start_date=20220823T115840, end_date=20220823T115840
[2022-08-23 11:58:40,902] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-23 11:58:40,959] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
